src/styles/_header.scss

```diff
@@ -0,0 +1,62 @@
+@use 'variables';
+@use 'fonts';
+
+.header {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+
+  height: variables.$headerHeight;
+  padding: 0 50px;
+
+  &__img {
+    width: 40px;
+    height: 40px;
+    text-align: center;
+  }
+
+  &__nav {
+    display: flex;
+    justify-content: flex-end;
+    align-items: stretch;
+    height: variables.$headerHeight;
+    font-size: 12px;
+  }
+
+  &__link {
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    color: variables.$mainColor;
```

Comment:
Ensure that `variables.$mainColor` is defined in your 'variables' file. This variable should specify the main color used for text and other elements.